src/ResourceManager/EventHub/Commands.EventHub/Models/PSEnumEncodingCaptureDescription.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.Azure.Commands.EventHub.Models
     /// <summary>
     /// Defines values for EncodingCaptureDescription.
     /// </summary>
-    public enum EnumEncodingCaptureDescription
+    public enum PSEnumEncodingCaptureDescription
```

Comment:
Enums should not have a PS prefix.  Please revert and change in necessary places.

Reply:
resolved